tests/jballerina-unit-test/src/test/resources/test-src/error/error_test.bal

```diff
@@ -199,7 +197,7 @@ function testGenericErrorWithDetailRecord() returns boolean {
     int detailStatusCode = 123;
     error e = error(reason, message = detailMessage, statusCode = detailStatusCode);
     string errReason = e.message();
-    map<value:Cloneable> errDetail = e.detail();
+    map<error:Cloneable> errDetail = e.detail();
```

Comment:
Do we have test cases, covering `value:Cloneable` ?

Reply:
Cloneable is used in many other test areas and subsequent products of statements.